configure.ac

```diff
@@ -78,6 +80,10 @@ PKG_CHECK_MODULES(GLIB260, glib-2.0 >= 2.60,
                   [AC_DEFINE(GLIB_VERSION_MIN_REQUIRED, GLIB_VERSION_2_60, [Ignore massive GTimeVal deprecation warnings in 2.62])],
                   [true])
 
+PKG_CHECK_MODULES(EPOLL_SHIM, epoll-shim, [true], [true])
```

Comment:
This shouldn't be necessary at all, looks like the `sys/eventfd.h` include is dead code, eventfd is not actually used here